spec/rubocop/cop/rspec/void_expect_spec.rb

```diff
@@ -44,4 +44,62 @@
       end
     RUBY
   end
+
+  it 'ignores unrelated expect method in an it block' do
+    expect_no_offenses(<<~RUBY)
+      it 'something' do
+        MyObject.expect
+      end
+    RUBY
+  end
+
+  it 'rejects bare `expect` in an it block' do
+    expect_offense(<<~RUBY)
+      it 'something' do
+        expect
+        ^^^^^^ Do not use `expect()` without `.to` or `.not_to`. Chain the methods or remove it.
+      end
+    RUBY
+  end
+
+  context 'when expect has no parent node' do
+    it 'still registers an offense' do
```

Comment:
This sounds weird, and doesn’t look correct at a glance. I believe the cop should not flag a top-level `expect`, and such statement is not valid RSpec, and should not have a spec for this case.